include/modem/pdn.h

```diff
@@ -54,6 +54,42 @@ struct pdn_pdp_opt {
 	uint8_t secure_pco;
 };
 
+/**
+ * @brief PDN PDP context information structure.
+ *
+ * This structure holds information about the PDN PDP context.
+ */
+struct pdn_pdp_context_info {
```

Comment:
Name and descriptions should include the dynamic to stay consistent.

```suggestion
struct pdn_pdp_context_dynamic_info {
```

Reply:
Ok